### PR TITLE
Migrate project to serverless framework

### PR DESCRIPTION
**Pull request by Omicron7**
Upgrade to nodejs 10 and use webpack.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+service: okta-hooks
```
Comment (soberstadt) [inline serverless.yml]: What does this do?

**Review by soberstadt — APPROVED** [review]
This all looks good to me. Do we have a way of testing that the deploy here works? I could probably do a QA after this deploys.

File: serverless.yml (hunk 1 above)
Comment (Omicron7) [reply serverless.yml]: That sets the name for all resources created in AWS.

**Comment by Omicron7** [comment]
The existing lambda will still exist after we deploy this, So you can hit the new one to test it and when it's working, we can delete the old one.

**Pull request merged**